components/ipfs/ipfs_utils.cc

```diff
@@ -134,17 +135,25 @@ bool IsIpfsDisabledByPolicy(PrefService* prefs) {
          !prefs->GetBoolean(kIPFSEnabled);
 }
 
+bool HasIPFSOrIPNSPath(const GURL& gurl) {
+  return gurl.is_valid() && (HasIPFSPath(gurl) || HasIPNSPath(gurl));
+}
+
 bool HasIPFSPath(const GURL& gurl) {
   const auto& path = gurl.path();
-  return gurl.is_valid() && ((path.find("/ipfs/") != std::string::npos) ||
-                             (path.find("/ipns/") != std::string::npos));
+  return gurl.is_valid() && base::Contains(path, "/ipfs/");
```

Comment:
shouldn't we check the position of this to make sure it's the first element in the path?

Reply:
Fixed